.github/workflows/run_periodic_tests.yml

```diff
@@ -92,7 +92,7 @@ jobs:
         if: matrix.os != 'windows-latest'
         run: python -m nox -s pybamm-requires
 
-      - name: Run unit tests for GNU/Linux with Python 3.8, 3.9, 3.10, and 3.12; and for macOS and Windows with all Python versions
+      - name: Run unit tests for GNU/Linux, macOS, and Windows with all Python versions
         if: (matrix.os == 'ubuntu-latest' && matrix.python-version != 3.11) || (matrix.os != 'ubuntu-latest')
```

Comment:
Now we should be able to run coverage on 3.12

Reply:
Lets do that elsewhere